soroban-test-wasms/wasm-workspace/Cargo.toml

```diff
@@ -96,8 +96,7 @@ wasmi = { package = "soroban-wasmi", git = "https://github.com/stellar/wasmi", r
 
 soroban-sdk = { git = "https://github.com/stellar/rs-soroban-sdk", rev = "1d1c866d" }
 soroban-sdk-macros = { git = "https://github.com/stellar/rs-soroban-sdk", rev = "1d1c866d" }
-# TODO: Update this hash after the SDK PR merges: https://github.com/stellar/rs-soroban-sdk/pull/670.
-soroban-spec = { git = "https://github.com/stellar/rs-soroban-sdk", rev = "0f3fc6db" }
+soroban-spec = { git = "https://github.com/stellar/rs-soroban-sdk", rev = "ad95e20c" }
```

Comment:
fyi @graydon I finally hit a case where I had to update the circular refs in the env repo to point to a pr in the sdk repo.